src/neptune/attributes/atoms/integer.py

```diff
@@ -45,5 +50,17 @@ def assign(self, value: typing.Union[IntegerVal, float, int], *, wait: bool = Fa
         if not isinstance(value, IntegerVal):
             value = IntegerVal(value)
 
+        if Integer.MIN_32_BIT_INT < value.value < Integer.MAX_32_BIT_INT:
+            logger.warning(
+                "WARNING: The value you're trying to log is outside the range of 32-bit integers "
+                "(%s to %s) and will be skipped. "
+                "We'll support 64-bit integers in the future. "
+                'For now, try logging the value as a float instead: run["field"] = float(%s)',
+                Integer.MIN_32_BIT_INT,
+                Integer.MAX_32_BIT_INT,
+                value.value,
+            )
+            return
```

Comment:
`return None` is more convenient.

Reply:
Is it, really? For function which is meant to be a procedure? Seems misleading.

Reply:
Ok. ChatGPT agrees with you :P.